object_intern_test.go

```diff
@@ -315,8 +315,7 @@ func TestIncRefCount(t *testing.T) {
 			t.Error("Failed to AddOrGet: ", b)
 			return
 		}
-		// need to compress b just as is done in the actual AddOrGet method
-		results[string(oi.compress(b))] = ret
+		results[string(b)] = ret
```

Comment:
what was this change about?

Reply:
Oh, it just doesn't actually matter as we never use the key, we only iterate over it and look at values